squirrel.frontier/src/test/java/org/dice_research/squirrel/queue/domainbased/MongoDBDomainQueueTest.java

```diff
@@ -162,4 +232,84 @@ public void deleteUris() throws Exception {
         assertEquals(0, mongodbQueue.length());
         mongodbQueue.close();
     }
+
+    @Test
+    public void testMakeURIsKeywise() throws URISyntaxException {
```

Comment:
This test method should not be part of this test class since the grouping of URIs is done by a different class than the queue.